worker/workercmd/guardian.go

```diff
@@ -51,25 +45,9 @@ func (cmd *WorkerCommand) guardianRunner(logger lager.Logger) (ifrit.Runner, err
 		"--no-image-plugin",
 	}
 
-	for _, dnsServer := range cmd.Garden.DNSServers {
-		gdnServerFlags = append(gdnServerFlags, "--dns-server", dnsServer)
-	}
-
-	for _, denyNetwork := range cmd.Garden.DenyNetworks {
-		gdnServerFlags = append(gdnServerFlags, "--deny-network", denyNetwork)
-	}
-
-	if cmd.ContainerNetworkPool != "" {
-		gdnServerFlags = append(gdnServerFlags, "--network-pool", cmd.ContainerNetworkPool)
-	}
-
-	if cmd.Garden.MaxContainers != 0 {
-		gdnServerFlags = append(gdnServerFlags, "--max-containers", strconv.Itoa(cmd.Garden.MaxContainers))
-	}
-
-	gdnServerFlags = append(gdnServerFlags, detectGardenFlags(logger)...)
+	gdnFlags = append(gdnFlags, detectGuardianFlags(logger)...)
```

Comment:
Have we ever talked about if we want to automatically detect and pass on flags for `containerd` as well?

Reply:
Good question. I haven't had the conversation with anyone. However, we allow for a config file to be passed (`CONCOURSE_CONTAINERD_CONFIG`) which I feel is a reasonable option, and easier for us to maintain. There also aren't that many options on the binary.

![Screen Shot 2020-07-29 at 9 41 23 AM](https://user-images.githubusercontent.com/5854091/88807414-af47ee00-d17f-11ea-9e15-9dd13d29a900.png)